.github/workflows/test_and_build.yml

```
@@ -142,6 +142,7 @@ jobs:
           # Consider removing old versions when they become problematic or very old (>=2 years).
           nxver=$(python -c 'import random ; print(random.choice(["=2.7", "=2.8", "=3.0", ""]))')
           yamlver=$(python -c 'import random ; print(random.choice(["=5.4", "=6.0", ""]))')
+          sparsever=$(python -c 'import random ; print(random.choice(["=0.12", "=0.13", ""]))')
```

Accept this suggestion if you haven't already changed it locally :)
```suggestion
          sparsever=$(python -c 'import random ; print(random.choice(["=0.13", ""]))')
```